strawberryfields/apps/qchem/utils.py

```diff
@@ -96,3 +96,59 @@ def duschinsky(
     delta = np.array(d @ l0_inv)
 
     return U, delta
+
+
+def read_gamess(file) -> Tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray]:
+    r"""Reads molecular data from the output file generated by the GAMESS quantum chemistry package.
+
+    This function extracts the atomic coordinates, atomic masses, vibrational frequencies,
+    and normal modes of a molecule from the output file of a vibrational frequency calculation
+    performed with the GAMESS quantum chemistry package.
+
+    **Example usage:**
+
+    >>> r, m, w, l = read_gamess('../BH_frq.out')
+    >>> r
+    array([0.000000, 0.000000, 0.000000, 1.253604, 0.000000, 0.000000]
+
+    Args:
+        file: name and the path of the GAMESS output file
+
+    Returns:
+        tuple[array, array, array, array]: atomic coordinates, atomic masses, normal mode
+        frequencies, normal modes
+    """
+    with open(file, "r") as f:
+
+        r = []
+        m = []
+        w = []
+        l = []
+
+        for line in f:
+
+            if "INPUT CARD> $data" in line or "INPUT CARD> $DATA" in line:
+                line = [next(f) for _ in range(3)][-1]
+                while "end" not in line and "END" not in line:
+                    r.append(np.array(line.rstrip().split()[-3:], float))
+                    line = next(f).rstrip()
+
+            if "ATOMIC WEIGHTS" in line:
```

Comment:
Probably it is convenient to raise an error if the output group is not found and print an error message. For example, what happens if the output file does not correspond to a frequency calculation?.